packages/esm-form-entry-app/src/app/form-schema/form-schema.service.ts

```diff
@@ -6,18 +6,31 @@ import { FormResourceService } from '../openmrs-api/form-resource.service';
 import { FormSchemaCompiler } from '@openmrs/ngx-formentry';
 import { LocalStorageService } from '../local-storage/local-storage.service';
 import { FormSchema, Questions } from '../types';
+import { TranslateService } from '@ngx-translate/core';
 
 @Injectable()
 export class FormSchemaService {
   constructor(
     private formsResourceService: FormResourceService,
     private localStorage: LocalStorageService,
     private formSchemaCompiler: FormSchemaCompiler,
+    private translate: TranslateService,
   ) {}
 
   public getFormSchemaByUuid(formUuid: string, cached: boolean = true): Observable<FormSchema> {
     const cachedCompiledSchema = this.getCachedCompiledSchemaByUuid(formUuid);
+    const currentLang = (window as any).i18next?.language.substring(0, 2).toLowerCase() || 'en';
```

Comment:
Instead of treating `window` as `any` I recommend to extend the Window interface with `i18next`:

```ts
declare global {
  interface Window {
    i18next?: {
      language: string;
    };
  }
}
```

(Ideally the definition of the `i18next` property / instance is taken from `i18next` directly - I think its called something like `i18n`.)